extension/observer/ecsobserver/metadata.yaml

```diff
@@ -5,3 +5,5 @@ status:
   stability:
     beta: [extension]
   distributions: [contrib, aws, splunk, sumo]
+  codeowners:
+    active: [dmitryax @rmfitzpatrick]
```

Comment:
Should there be an `@`? Perhaps it's missing a comma?

Reply:
Good catch!

Reply:
Fixed.